workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/context/SolutionContext.tsx

```diff
@@ -411,7 +415,12 @@ const reducer = (state: SolutionInitialState, action: AppActions) => {
 					),
 					description: _product.description.en_US,
 					file: {
+						changed: false,
+						fileName: appIcon?.title.en_US as string,
```

Comment:
The appIcon can be undefined causing runtime error

```suggestion
						fileName: appIcon?.title?.en_US as string,
```